src/main/java/com/quantumlytangled/gravekeeper/util/SoulboundHandler.java

```diff
@@ -25,13 +25,22 @@ public static boolean isSoulbound(@Nonnull final ItemStack itemStack) {
     // check for NBT tags
     final NBTTagCompound tagCompound = itemStack.getTagCompound();
     if (tagCompound != null) {
-      for (final Map.Entry<String, Boolean> tag : GraveKeeperConfig.SOULBOUND_TAGS.entrySet()) {
-        if (tagCompound.hasKey(tag.getKey())
-            && tagCompound.getBoolean(tag.getKey()) == tag.getValue()) {
+      for (final NBTTagCompound tagEntry : GraveKeeperConfig.SOULBOUND_TAGS) {
```

Comment:
'entry' refers to map entries, here it's more a tagElement of the list or the tagExpected value.